spark/v3.2/build.gradle

```diff
@@ -64,8 +64,6 @@ project(':iceberg-spark:iceberg-spark-3.2') {
     compileOnly("org.apache.spark:spark-hive_2.12:${sparkVersion}") {
       exclude group: 'org.apache.avro', module: 'avro'
       exclude group: 'org.apache.arrow'
-      // to make sure io.netty.buffer only comes from project(':iceberg-arrow')
```

Comment:
Do we remove this on purpose? I think it was added recently.

Reply:
We should not remove this. It messed up when I merge multiple commits.